.travis.yml

```diff
@@ -29,7 +29,12 @@ before_install:
   - conda info -a
 
 install:
+  #- conda install anaconda=2.4
+  #- conda create --name testoceanmodes --file oceanmodes-exported-packages.txt
+  #- source activate testoceanmodes
   - conda create -q -n test-environment python=$TRAVIS_PYTHON_VERSION $CONDA_PACKAGES
+  - conda install scipy numpy jupyter ipython matplotlib nomkl scikit-learn numexpr --name=test-environment
+  - conda remove mkl mkl-service --name=test-environment
```

Comment:
@roxyboy: these packages need to go into the variable CONDA_PACKAGES, which is set in line 16